xml_converter/generators/code_generator.py

```diff
@@ -359,13 +388,55 @@ def build_example(self, type: str, applies_to: List[str], xml_field: str, exampl
 
         return example
 
+    ############################################################################
+    # Generate Node Types
+    #
+    # This will output code for a single category of nodes.
+    ############################################################################
+    def generate_node_types(self, metadata: Any, attributenames: Dict[str,str],page: str) -> Tuple[List[str],List[str]]:
+        
+        field_rows: List[str] = []
```

Comment:
`row` does not have much meaning here as it was originally from a table row. So we can just call it something like `class_variables`, `class_fields`, `node_variables`, or `field_variables`.

Reply:
changed to `attribute_variables`